### PR TITLE
Tech and Content Review - Report Status

### PR DESCRIPTION
**Pull request by KellyVance**
Report status vocabulary for lodgement portal.
Very hypothetical terms. 
Please comment as needed.

### CODE REVIEW

File: vocabularies/report-status.ttl [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    skos:definition "The report is in-progress and saved in the lodgement portal awaiting submission."@en ;
+    skos:inScheme rptst:conceptScheme ;
+    skos:prefLabel "In-Progress"@en ;
+    skos:topConceptOf rptst:conceptScheme .
+
+rptst:quarantined a skos:Concept ;
+    skos:definition "The report is held by the department for examination prior to submission being allowed"@en ;
+    skos:inScheme rptst:conceptScheme ;
+    skos:prefLabel "Quarantined"@en ;
+    skos:topConceptOf rptst:conceptScheme .
```
Comment (GSQ-AI) [inline vocabularies/report-status.ttl]: rptst:screening a skos:Concept ;
    skos:definition "The report is held by the department for examination prior to submission being allowed"@en ;
    skos:inScheme rptst:conceptScheme ;
    skos:prefLabel "Screening"@en ;
    skos:topConceptOf rptst:conceptScheme .

**Comment by DavidCrosswellGSQ** [comment]
I'd be happy for this to have the concepts:
in progress
submitted
abandoned - if not submitted within 90 days, the system marks as abandoned and deletes the S3 resources

Not sure why quarantined is needed?

**Comment by KellyVance** [comment]
@DavidCrosswellGSQ  @GSQ-AI 
I have added an abandoned concept.
Screening was purely hypothetical. e.g reports from company X are consistently bad (maybe data accuracy, rather than validity which the system would reject). We select a report to hold in a quarantined state for screening.

**Comment by GSQ-AI** [comment]
My approval stands, following inclusion of "Abandoned".

**Comment by DavidCrosswellGSQ** [comment]
Ok - that makes sense.

**Pull request merged**